CLogger/include/Utils/Utils.h

```diff
@@ -43,22 +17,17 @@ namespace LogUtils
 	/// </example>
 	inline std::string GetCurrentTimeString()
 	{
-		constexpr auto special_year = 1900;
 		const auto time = std::make_unique<tm>();
 		auto current_time = std::chrono::system_clock::to_time_t
 		(std::chrono::system_clock::now());
 
 		localtime_s(time.get(), &current_time);
 
-		const auto day   = AddZero(time->tm_mday);
-		const auto month = AddZero(time->tm_mon + 1);
-		const auto year  = AddZero(time->tm_year + special_year);
-		const auto hour  = AddZero(time->tm_hour);
-		const auto min   = AddZero(time->tm_min);
-		const auto sec   = AddZero(time->tm_sec);
+		std::stringstream ss;
+		ss.imbue(std::locale());
+		ss << std::put_time(time.get(), "%c");
```

Comment:
I think, it's better to use [std::strftime](https://en.cppreference.com/w/cpp/chrono/c/strftime) here.

Reply:
They work the same, but `std::strftime` is C-legacy thing, so I need to create a `char` buffer before I use it. What about speed: 700-800 microseconds both variants. In this case more useful for me `std::put_time`.